server/src/main/java/org/elasticsearch/index/mapper/MapperFeatures.java

```diff
@@ -52,6 +52,7 @@ public class MapperFeatures implements FeatureSpecification {
     static final NodeFeature IGNORED_SOURCE_FIELDS_PER_ENTRY = new NodeFeature("mapper.ignored_source_fields_per_entry");
     public static final NodeFeature MULTI_FIELD_UNICODE_OPTIMISATION_FIX = new NodeFeature("mapper.multi_field.unicode_optimisation_fix");
     static final NodeFeature PATTERN_TEXT_RENAME = new NodeFeature("mapper.pattern_text_rename");
+    public static final NodeFeature ON_DISK_RESCORING = new NodeFeature("mapper.vectors.on_disk_rescoring");
```

Comment:
Should we just add the option to all HNSW vector formats, so it's at least accepted as a valid option, even if it's not used yet?

Reply:
I've added this in 5535ba1352c (it can easily be reverted if not needed now)